service-statistics-pxweb/src/main/java/fi/nls/oskari/control/statistics/plugins/pxweb/PxwebStatisticalDatasourcePlugin.java

```diff
@@ -179,4 +185,31 @@ private String createUrl(String baseUrl, String pathId) {
         }
         return baseUrl;
     }
+
+    private ValueProcessor getValueProcessor(DatasourceLayer layer) {
+        if(layer.getConfig() == null) {
```

Comment:
This check is not unnecessary since `DatasourceLayer#getConfig(String key)` does the null check for you.

Reply:
True, I used getConfig().optString() before in the code and this got left after the change. Good catch.